docs/lazy-loading.md

```diff
@@ -45,3 +45,31 @@ The radius needs to be tuned to each size individually. Facebook uses about 200
 Gaussholder includes a CLI command to help you tune the radius: pick a representative attachment or image file and use `wp gaussholder check-size <id_or_image> <radius>`. Adjust the radius until you get to roughly 200B, then check against other attachments to ensure they're in the ballpark.
 
 Note: changing the radius requires regenerating the placeholder data. Run `wp gaussholder process-all --regenerate` after changing radii or adding new sizes.
+
+## Lazy Loading iFrames
+
+iFrames are lazy-loaded by default using the [browser-level `loading` attribute](https://html.spec.whatwg.org/multipage/iframe-embed-object.html#attr-iframe-loading). Only `iframe` tags with both `width` and `height` attributes present will be lazy-loaded to avoid a negative impact on layout shifting. oEmbeded iframe content within `the_content`, `the_excerpt` and `widget_text_content` will have the `loading="lazy"` tag added by default where the web service has provided a `width` and `height` attribute.
```

Comment:
```suggestion
Inline frames are lazy-loaded by default using the [browser-level `loading` attribute](https://html.spec.whatwg.org/multipage/iframe-embed-object.html#attr-iframe-loading). Only `<iframe>` tags with both `width` and `height` attributes present will be lazy-loaded to avoid a negative impact on layout shifting. Embedded `<iframe>` tags provided via oEmbed in content run through `the_content`, `the_excerpt` or `widget_text_content` filters will have the `loading="lazy"` attribute added by default where the web service has provided a `width` and `height` attribute.
```

I haven't seen the `iFrame` capitalisation anywhere before but we should make it consistent. On MDN for example in plain english they're always referred to as inline frames or `<iframe>`. The "i" should at least be capitalised in case Apple decide to sue us 😛

Reply:
A quick search shows a bunch of articles about iframes with a capital F but none of those are from respected documentation sources (except Microsoft docs). W3Schools actually capitalizes the "I" which is blasphemy. 😂

![Screen Shot 2021-04-15 at 10 30 14 AM](https://user-images.githubusercontent.com/991511/114905088-cb56f280-9dd5-11eb-984e-8218b186e442.png)

This might go back to my memory of HTML 2.0 days where eVerYthInG waS camElCasE. "Inline frames" and the actual tag are _much_ better. 👍